### PR TITLE
[WIP] Azure DataBus 3.0

### PR DESCRIPTION
**Pull request by SeanFeldman**
- Doco update
- Upgrade guide from v2 to v3
- Snippets for v3
- Sample for v3

### CODE REVIEW

**Comment by SeanFeldman** [comment]
WIP till a release build is on myget

**Comment by yvesgoeleven** [comment]
Sample tested against myget build, it works

File: nservicebus/messaging/databus/azure-blob-storage.md [hunk 1]
```diff
@@ -22,19 +22,17 @@ snippet: AzureDataBus
...
 
-WARN: the built-in method uses continuous blob scanning which can add to the cost of the storage operations. It is **not** recommended for multiple endpoints that are scaled out. If this method is not used, be sure to disable the built-in cleanup by setting the `CleanupInterval` to `0`.
+WARN: the built-in method uses continuous blob scanning which can add to the cost of the storage operations. It is **not** recommended for multiple endpoints that are scaled out. If this method is not used, be sure to disable the built-in cleanup by setting the `CleanupInterval` to `0`. In versions 3 and above built-in cleanup is disabled by default.
 
 ### Using an Azure Durable Function
 
-Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to see how to use a durable function to clean up attachments. Be sure to [disable blob cleanup](#configuration-disabling-blob-cleanup) first.
+Review the [sample](/samples/azure/blob-storage-databus-cleanup-function/) to see how to use a durable function to clean up attachments.
```
Comment (yvesgoeleven) [inline nservicebus/messaging/databus/azure-blob-storage.md]: Are you sure you want to remove `Be sure to [disable blob cleanup](#configuration-disabling-blob-cleanup) first.` completely? Perhaps keep it and mention `for version 2 and earlier`?

Comment (SeanFeldman) [reply nservicebus/messaging/databus/azure-blob-storage.md]: Addressed by https://github.com/Particular/docs.particular.net/pull/4361/commits/708d783121b9c13ecbf8faeb388434a62371b43d

**Pull request merged**